### ISSUE TITLE
Target Electron 7

### ISSUE THREAD

**Issue by splix** [grpc/grpc-node#1084]
### Problem description
gRPC native module doesn't compile against Electron 7.0, which is the latest _stable version_ of Electron.

### Reproduction steps
```
mkdir grpc-electron-7-error-demo
cd grpc-electron-7-error-demo

echo "{}" > package.json
npm install grpc@1.24.1
cd node_modules/grpc

export npm_config_target=7.0.0
export npm_config_arch=x64
export npm_config_target_arch=x64
export npm_config_disturl=https://electronjs.org/headers
export npm_config_runtime=electron
export npm_config_build_from_source=true
HOME=~/.electron-gyp npm install
```

Fails with:
```
../ext/call.cc:104:53: error: no matching member function for call to 'Get'
    Local<String> current_key(Nan::To<String>(keys->Get(i)).ToLocalChecked());
                                              ~~~~~~^~~
/HOME/.electron-gyp/Library/Caches/node-gyp/7.0.0/include/node/v8.h:3502:43: note: candidate function not viable: requires 2 arguments, but 1 was provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                          ^
/HOME/.electron-gyp/Library/Caches/node-gyp/7.0.0/include/node/v8.h:3505:43: note: candidate function not viable: requires 2 arguments, but 1 was provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                          ^

...

../ext/call.cc:681:27: error: no matching member function for call to 'Get'
    if (!op->ParseOp(obj->Get(type), &ops[i])) {
                     ~~~~~^~~
/HOME/.electron-gyp/Library/Caches/node-gyp/7.0.0/include/node/v8.h:3502:43: note: candidate function not viable: requires 2 arguments, but 1 was provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                          ^
/HOME/.electron-gyp/Library/Caches/node-gyp/7.0.0/include/node/v8.h:3505:43: note: candidate function not viable: requires 2 arguments, but 1 was provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                          ^

```

### Environment
 - OS name, version and architecture: macOS (but same issue on Ubuntu too)
 - Node version: node v12.12.0 + electron v7.0.0
 - Node installation method: nvm
 - If applicable, compiler version: Apple clang version 11.0.0 (clang-1100.0.33.8), x86_64-apple-darwin19.0.0
 - Package name and version: gRPC@1.24.1

**Comment by splix** [comment]
#1089

**Comment by murgatroid99** [comment]
I have now published grpc 1.24.2 with support for Electron 7.
